tests/src/hiprtc/saxpy.cpp

```diff
@@ -83,7 +84,7 @@ int main()
     size_t codeSize;
     hiprtcGetCodeSize(prog, &codeSize);
 
-    vector<char> code(codeSize);
+    std::vector<char> code(codeSize);
```

Comment:
This should be unnecessary since we already have a using namespace std on line 60.

Reply:
Removed the std:;, since whole namespace has been imported.